pyfar/signals/deterministic.py

```diff
@@ -194,6 +211,102 @@ def linear_sweep_time(n_samples, frequency_range, n_fade_out=90, amplitude=1,
     return signal
 
 
+def linear_sweep_freq(
+        n_samples, frequency_range, start_margin, stop_margin, fade_in=None,
+        fade_out=None, butterworth_order=8, sampling_rate=44100):
+    """
+    Generate single channel sine sweep with linearly increasing frequency.
+
+    Sine sweep sweep synthesis according to [#]_.
```

Comment:
```suggestion
    Sine sweep synthesis according to [#]_.
```
Or do you want the "sweep sweep"? Same thing for the exponential sweep.